onboarding/main.go

```diff
@@ -0,0 +1,161 @@
+package main
+
+import (
+	"context"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/pem"
+	"os"
+
+	openshiftv1 "github.com/openshift/api/template/v1"
+	ocsv1 "github.com/red-hat-storage/ocs-operator/v4/api/v1"
+	ocsv1alpha1 "github.com/red-hat-storage/ocs-operator/v4/api/v1alpha1"
+	"github.com/red-hat-storage/ocs-operator/v4/controllers/util"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	errruntime "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	apiruntime "k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/klog"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+)
+
+const (
+	onboardingTicketPublicKeySecret  = "onboarding-ticket-public-key"
+	onboardingTicketPrivateKeySecret = "onboarding-ticket-private-key"
+)
+
+var (
+	ctx                   = context.Background()
+	serverNamespace       = os.Getenv(util.WatchNamespaceEnvVar)
```

Comment:
Just curious do we use WatchNamespaceEnvVar instead of OperatorNamespaceEnvVar?

Reply:
We can use both i guess but right it makes more sense to use OperatorNamespaceEnvVar